Lombiq.Tests.UI.Samples/Lombiq.Tests.UI.Samples.csproj

```diff
@@ -12,6 +12,10 @@
     <None Remove=".htmlvalidate.json" />
     <None Remove="xunit.runner.json" />
   </ItemGroup>
+  
```

Comment:
This still isn't addressed. See the following for what I'm talking about:

- `HtmlValidationOrchardCoreUITestExecutorConfigurationExtensions`
- `HtmlValidationAssertionException`
- `HtmlValidationConfiguration`
- `HtmlValidationUITestContextExtensions`

The same happens for accessibility:

![image](https://user-images.githubusercontent.com/1976647/172933627-da9b01f0-6bf5-4932-8c25-cf00b04e93d7.png)

Reply:
I still don't see this being addressed. Do you have a question related to this? In any case, please don't just ignore my repeated remarks.

Reply:
Now its clear..., but I can't see how can we validate on every page change without setting the threshold to an extremely high level.
[Here](https://lombiq.atlassian.net/browse/OSOE-111#:~:text=Lombiq.Tests.UI.Samples.Tests.BasicVisualVerificationTests.VerifyHomePage(browser_%2DChrome)) you can see my full page test results on different environments.

Reply:
Do I understand correctly that due to the TC and GH differences we'd need something like `meanErrorPercentageThreshold = 50`?

Since your link doesn't open it for me, I assume we're talking about [this comment](https://lombiq.atlassian.net/browse/OSOE-111?focusedCommentId=65574) and the section under "Lombiq.Tests.UI.Samples.Tests.BasicVisualVerificationTests.VerifyHomePage(browser_-Chrome)". BTW nice details of the measurements there!

Reply:
I see, fair point, let's not do this then.